src/game/mario_actions_automatic.c

```diff
@@ -725,7 +725,9 @@ s32 act_in_cannon(struct MarioState *m) {
             marioObj->oMarioCannonInputYaw -= (s16)(m->controller->stickX * 10.0f);
 
             m->faceAngle[0] = CLAMP(m->faceAngle[0], 0, DEGREES(80));
+    #ifndef UNRESTRICTED_CANNON_AIMING
             marioObj->oMarioCannonInputYaw = CLAMP(marioObj->oMarioCannonInputYaw, -0x4000, 0x4000);
```

Comment:
```suggestion
            marioObj->oMarioCannonInputYaw = CLAMP(marioObj->oMarioCannonInputYaw, -DEGREES(90), DEGREES(90);
```
Might as well do this too. Non-blocking.